site/app/libraries/database/DatabaseQueries.php

```diff
@@ -7355,6 +7355,15 @@ public function getGradeableAccessTeam(
         return $this->course_db->rows();
     }
 
+    public function getUserGroups(string $user_id): array {
```

Comment:
What is the purpose of this function?  It should only grab the group for courses in which that user has instructor access?

Reply:
I added a check to make sure the user is an instructor of that course. The purpose of this function is to feed to the front end for faculty to choose the group they want to use.